maps/chronosphere/comfylatron.lua

```diff
@@ -388,7 +599,7 @@ local function spawn_comfylatron(surface_index, x, y)
 	if not objective.comfylatron_habitat then
 		objective.comfylatron_habitat = {
 			left_top = {x = -32, y = -192},
-			right_bottom = {x = 32, y = 192}
+			right_bottom = {x = 32, y = -24} -- stops comfytron getting stuck in chests

```

Comment:
nonsense. that excludes whole wagon

Reply:
it limits it to upper half of wagon, she spawns fine

she spawned in the middle of the eight central chests once to me then was stuck :(

Reply:
desync function makes him respawn after some time. is fine